auslib/admin/views/permissions.py

```diff
@@ -26,6 +26,21 @@ def get(self):
         return jsonify(dict(users=users))
 
 
+class CurrentUserView(AdminView):
+    """Returns all of the details about the logged in user. The UI needs this
+    method to know things about the current user because it does not have
+    access to REMOTE_USER, so it cannot query directly by name."""
+
+    def get(self, username):
+        if username == "current":
+            username = request.environ.get('REMOTE_USER', request.environ.get("HTTP_REMOTE_USER"))
+        permissions = dbo.permissions.getUserPermissions(username)
```

Comment:
Implementing this in the most robust way (at the database layer) requires an interface change to AUSTable.select. I'm not sure we should scope creep this PR to do that, so I did a quick implementation in this endpoint, and filed https://bugzilla.mozilla.org/show_bug.cgi?id=1340167 to move it to the database layer. I'd like to get it fixed soon, but the status quo allows this already, so it doesn't seem extremely pressing.